src/component/axis/CartesianAxisView.ts

```diff
@@ -135,6 +136,30 @@ const axisElementBuilders: Record<typeof selfBuilderAttrs[number], AxisElementBu
             tickModel: splitLineModel
         });
 
+        const axisScale = axis.scale as IntervalScale;
+        if (ticksCoords.length > 2 && axisScale.getInterval) {
```

Comment:
What if `ticksCoords.length` is `2` but the data range is not equal to the ticks' values?

Reply:
When ticksCoords length is 2,  the splitLine is not expected to be hidden

Reply:
Why? For example, when data range is 11~19 and ticksCoords is [10, 20], I believe there should be no split line, right?